Jenkinsfile_CNP

```diff
@@ -34,6 +34,17 @@ def secrets = [
         secret('ccd-restricted-cross-case-type-caseworker-email', 'CCD_RESTRICTED_CROSS_CASE_TYPE_CASEWORKER_EMAIL'),
         secret('ccd-restricted-cross-case-type-caseworker-password', 'CCD_RESTRICTED_CROSS_CASE_TYPE_CASEWORKER_PASSWORD'),
         secret('ccd-api-gateway-oauth2-client-secret', 'OAUTH2_CLIENT_SECRET'),
+
+        //Required for FTA test RDM-6943
+        secret('ccd-caseworker-solicitor-one-password', 'CCD_CASEWORKER_AUTOTEST_SOLICITOR_ONE_PASSWORD'),
+        secret('ccd-caseworker-solicitor-two-password', 'CCD_CASEWORKER_AUTOTEST_SOLICITOR_TWO_PASSWORD'),
+        secret('ccd-caseworker-solicitor-three-password', 'CCD_CASEWORKER_AUTOTEST_SOLICITOR_THREE_PASSWORD'),
+        secret('ccd-caseworker-one-password', 'CCD_CASEWORKER_ONE_AUTOTEST_PASSWORD'),
+        secret('ccd-caseworker-two-password', 'CCD_CASEWORKER_TWO_AUTOTEST_PASSWORD'),
+        secret('ccd-caseworker-three-password', 'CCD_CASEWORKER_THREE_AUTOTEST_PASSWORD'),
+        secret('ccd-citizen-two-password', 'CCD_CITIZEN_TWO_AUTOTEST_PASSWORD'),
+        secret('ccd-citizen-three-password', 'CCD_CITIZEN_THREE_AUTOTEST_PASSWORD'),
+        secret('ccd-solicitor-three-password', 'CCD_AUTOTEST_THREE_SOLICITOR_PASSWORD')
```

Comment:
could we please rename these like
_befta.caseworker.1_ --> **CCD_**_BEFTA_CASE_WORKER_1_**_PWD**